### PR TITLE
Add basic docusaurus structure

### PR DESCRIPTION
**Pull request by oscarpolanco**
Adding basic docs structure to the repo.

# Steps for testing the site locally:
- Go to the `website` folder
- Install all dependencies using `npm install`
- Start your local server using `npm run start`
- Test locally at [localhost](http://localhost:3000/)

# Steps to test the build:
- Go to the `website` folder
- Install all dependencies using `npm install`
- Start your local server using `npm run build`
- Check that now you got a `build` folder under the `website` directory

# First deploy process (manual):
- Create a local branch call `gh-pages` from `master`
- Go to the `website` folder
- Install packages using `npm install`
- Do a build using `npm build`
- go back to the `root` folder
- Add the files and commit using this command => `git add -f website/build/amphora-html/ && git commit -m"deploy files"`
- Push the static files using this command => `git  subtree push --prefix website/build/amphora-html origin gh-pages`
## Note:
- This Process I will do it when the` PR` is approved
- We only need the static files on the `gh-pages` branch and the commit history only will show docs update and important remember that the static files are part of the `.gitignore` that's why we add and push this way.
- Any suggestions to have a better first deploy process are more than welcome 😄 

# General Notes: 
- We still don't have the final design
- Leave some of the default files for future references
- We don't have the final content of the docs (I didn't add the version for this reason that will be another PR)

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+sidebar_label: Amphora HTML
+---
+
+---
+The HTML renderer for Clay components that use [Handlebars](http://handlebarsjs.com/) templates.
+
+---
+## Why?
+
+HTML rendering was controlled entirely by [Amphora](https://github.com/nymag/amphora) in v2.x, which meant that Amphora was responsible for a lot of heavy lifting. Furthermore, rendering was handled by a module called [Multiplex Templates](https://www.npmjs.com/package/multiplex-templates) which did some :crystal_ball: dark magic :sparkles: to synchronously render component templates. This required some affordances from Amphora which added to the weight of the middleware. By separating renderers out into separate modules which can be plugged into Amphora >v2.x we can create renderers for specific requirements (XML, Amp, etc.) _and_ the rendering process can be moved to separate machines to create more maintainable systems.
```
Comment (fxisco) [inline docs/intro.md]: We could add some style to `>v2.x` maybe bold to highlight this important info

File: docs/integration.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+---
+id: integration
+title: Integration
+sidebar_label: Integration
+---
+
+---
+## Basic Configuration
+
+First, ensure that you have a compatible version of Amphora installed (v3.x or greater) and require `amphora-html` at the from wherever you are running Amphora.
```
Comment (manuelurenah) [inline docs/integration.md]:
```suggestion
First, ensure that you have a compatible version of Amphora installed (v3.x or greater) and require `amphora-html` from wherever you are running Amphora.
```

File: docs/integration.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+You can add Amphora HTML plugins like this:
+
+```javascript
+amphoraHtml.addPlugins([
+  plugin1,
+  plugin2
+]);
+```
+
+One use case for Amphora HTMK plugins is to skip rendering of certain components depending on query parameters, which are available in `locals.query`. Amphora HTML plugins do not run when rendering for edit mode.
```
Comment (manuelurenah) [inline docs/integration.md]:
```suggestion
One use case for Amphora HTML plugins is to skip rendering of certain components depending on query parameters, which are available in `locals.query`. Amphora HTML plugins do not run when rendering for edit mode.
```

File: docs/intro.md (hunk 1 above)
Comment (oscarpolanco) [reply docs/intro.md]: Done!! Thanks!!

File: docs/integration.md (hunk 2 above)
Comment (oscarpolanco) [reply docs/integration.md]: Nice catch

**Pull request merged**